firefox-ios/firefox-ios-tests/Tests/ClientTests/Mocks/MockTest.swift

```diff
@@ -0,0 +1,6 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I think once we have tested this work, this file can be removed @yoanarios ?

Reply:
yes, I already remove this one and create a new one in Client target to test if danger fails the PR. The other file `NoCoverageClass` will be also removed after the test